.github/workflows/actions.yml

```diff
@@ -45,6 +42,61 @@ jobs:
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           path-to-lcov: _conbench-coverage-dir/coverage.lcov
+
+  # This is to confirm that commonly used developer commands still work.
+  dev-cmds:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
```

Comment:
The first time I realized that YML interprets `python-version: 3.10` (with no quotation marks) as the number equivalent to "3.1" I laughed out loud.

Reply:
Indeed rather surprising and ridiculous. This is the second or third time I run into this.

Thanks for looking!